modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1748,6 +1748,10 @@
 			<category name="Upgrade" />
 			<description name="Replace parameter in updateCommerceAddress method by other parameters list" />
 		</check>
+		<check name="UpgradeJavaUpdateFileEntryMethodCheck">
+			<category name="Upgrade" />
+			<description name="Fill in the new parameters of the method 'updateFileEntry' of 'DLAppLocalService' and 'DLAppLocalServiceUtil'" />
```

Comment:
@TamyrisTorres, please make sure to update this message depending of your answer in https://github.com/natocesarrego/liferay-portal/pull/1353#discussion_r1304207554?